Base/usr/share/man/man2/posix_spawn.md

```diff
@@ -0,0 +1,47 @@
+## Name
+
+`posix_spawn` - launch a new process
+
+## Synopsis
+
+```**c++
+#include <spawn.h>
+
+int posix_spawn(pid_t* pid, const char* file, const posix_spawn_file_actions_t*, const posix_spawnattr_t*, char* const argv[], char* const envp[]);
+int posix_spawnp(pid_t* pid, const char* file, const posix_spawn_file_actions_t*, const posix_spawnattr_t*, char* const argv[], char* const envp[]);
+```
+
+## Description
+
+Spawns a new process reading the binary `file`, passing `argv` as arguments to `main()` and setting `envp` as argument.
+
+Places the pid of the new process in `pid`.
+
+If `file` passed to `posix_spawn` is a relative path, it is resolved relative to the current working directory.
+
+If `file` passed to `posix_spawnp` is a relative path, it is resolved by searching `PATH`.
```

Comment:
Uh, perhaps we should name the argument something other than `file` then? `executable_path`, for example.

"By searching through directories specified in the `PATH` environment variable."